streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -918,12 +938,7 @@ public class StreamsConfig extends AbstractConfig {
                     Type.CLASS,
                     DefaultProductionExceptionHandler.class.getName(),
                     Importance.MEDIUM,
-                    DEFAULT_PRODUCTION_EXCEPTION_HANDLER_CLASS_DOC)
-            .define(PROCESSING_EXCEPTION_HANDLER_CLASS_CONFIG,
-                    Type.CLASS,
-                    LogAndFailProcessingExceptionHandler.class.getName(),
-                    Importance.MEDIUM,
-                    PROCESSING_EXCEPTION_HANDLER_CLASS_DOC)
+                    PRODUCTION_EXCEPTION_HANDLER_CLASS_DOC)
```

Comment:
Should this stay `DEFAULT_PRODUCTION_EXCEPTION_HANDLER_CLASS_DOC`?

Reply:
Because it is private already, aligned it with the config. I can change it back if you prefer to.

Reply:
Both are the same so it does not really matter too much. Just feels itchy to use the new `_DOC` variable for the old config. -- And the fact that the new one is private should not matter?